### PR TITLE
Added linting to test script

### PR DESCRIPTION
**Pull request by darkwebdev**
1. eslintrc is copied from this project
2. linting errors fixed

### CODE REVIEW

**Comment by platinumazure** [comment]
Hi @darkwebdev, thanks so much for your pull request.

I think it's a little too opinionated to assume that new plugin creators want to start with `eslint-config-eslint`. I'd much rather see a discussion around a possible new feature where we could allow users to decide if they want the lint configuration of their plugin to be based on any particular ESLint configuration. I do like the idea of ensuring that package.json has a lint and unit-test task. Would you be open to trimming this PR down to just that change and creating an issue around how we can integrate ESLint configurations into the plugin generator's prompts? Thanks!

File: plugin/index.js [hunk 1]
```diff
@@ -81,6 +81,7 @@ ESLintPluginGenerator.prototype.generate = function generate() {
     this.template("_plugin.js", "lib/index.js");
     this.template("_package.json", "package.json");
     this.template("_README.md", "README.md");
+    this.template("../../.eslintrc.yml", ".eslintrc.yml");
```
Comment (platinumazure) [inline plugin/index.js]: Per my comment on the issue, I think this is too opinionated. I'd be more interested in seeing if we could run `eslint --init` somehow in the process, or at least allowing the user to choose a different configuration.

File: plugin/templates/_package.json [hunk 2]
```diff
@@ -10,13 +10,16 @@
...
+    "lint": "eslint .",
+    "unit-test": "mocha tests --recursive",
+    "test": "npm run lint && npm run unit-test"
   },
   "dependencies": {
     "requireindex": "~1.1.0"
   },
   "devDependencies": {
     "eslint": "~3.9.1",
+    "eslint-config-eslint": "^3.0.0",
```
Comment (platinumazure) [inline plugin/templates/_package.json]: This also would be unnecessary/opinionated.

**Comment by platinumazure** [comment]
Also, I apologize for the AppVeyor error-- I haven't created an appveyor.yml file yet. I'm hoping to do that later today or tomorrow.

**Comment by darkwebdev** [comment]
Hey @platinumazure, thanks for the fast feedback!

I agree about `eslint --init`, I don't have an opinion about what rules to use, I'm only sure that should be some linting in the linting project. I'll make a different PR then.

**Pull request closed without merging**